docs/installing.md

```diff
@@ -136,3 +136,18 @@ $ tree -lla /home/ansible-dev/collections/ansible_collections
 └── .vscode
     └── extensions.json
 ```
+
+**Note:**
+
+The scaffolded collection includes a `hello_world` filter plugin, along with an integration test target for it, that can be run using `molecule` and `pytest-ansible`. This serves as an example for you to refer when writing tests for your Ansible plugins and can be removed when it is no longer required.
+
+To run the `hello_world` integration test, follow these steps:
+
+- Git initialize the repository containing the scaffolded collection with `git init`.
+- `pip install ansible-core molecule pytest-xdist pytest-ansible`.
+- Invoke `pytest` from collection root.
+
+It also comes equipped with Github Action Workflows that use [ansible-content-actions](https://github.com/marketplace/actions/ansible-content-actions) for testing and publishing the collection. For details on how to use these, please refer to the following:
+
+- [Using the testing workflow](https://github.com/ansible/ansible-content-actions/blob/main/docs/ci.md)
```

Comment:
These docs might not live here in the longer run, it's best it's referenced from the ansible-dev-tools repo.
They are the exactly the same docs!